netlify.toml

```diff
@@ -0,0 +1,12 @@
+# [build]
+# # Directory (relative to root of your repo) that contains the deploy-ready
+# # HTML files and assets generated by the build. If a base directory has
+# # been specified, include it in the publish directory path.
+# publish = "docs/export/"
+
+# # Default build command.
+# command = "yarn docs:build && yarn docs:export && yarn storybook:build && yarn storybook:export && yarn demo:build && yarn demo:export"
+
+[build.environment]
```

Comment:
restored so we keep the version definition (help make sure it works in the lowest target we support)